timed/subscription/filters.py

```diff
@@ -1,10 +1,17 @@
 from django_filters import FilterSet, NumberFilter
 
+from timed.projects.models import Project
+
 from . import models
 
 
 class PackageFilter(FilterSet):
-    customer = NumberFilter(field_name='billing_type__projects__customer')
+    customer = NumberFilter(method='filter_customer')
+
+    def filter_customer(self, queryset, name, value):
+        billing_types = (Project.objects.filter(customer=value)
```

Comment:
what is the reason to use braces here?

Reply:
the line is too long

Reply:
I guess this is matter of taste... but something like the following is properly more readable:
```python
        billing_types = Project.objects.filter(
            customer=value
        ).values(
            'billing_type'
        )
```